### PR TITLE
Add scala native

### PR DESCRIPTION
**Pull request by unilama**

### CODE REVIEW

File: src/compilers.coffee [hunk 1]
```diff
@@ -326,6 +326,12 @@
   target: 'Machine Code'
   type: 'Native'
   url: 'http://landinghub.visualstudio.com/visual-cpp-build-tools'
+,
+  name: 'Scala Natvie'
+  source: 'Scala'
+  target: 'Machine Code'
```
Comment (densh) [inline src/compilers.coffee]: I'd say it targets LLVM IR which transitively targets Machine Code.

Comment (unilama) [reply src/compilers.coffee]: changed to Intermediate and LLVM IR :) Thx for comment

File: src/compilers.coffee [hunk 2]
```diff
@@ -84,7 +84,7 @@
   target: 'LLVM IR'
   type: 'Intermediate'
   url: 'https://www.haskell.org/ghc/'
-,
+ ,
```
Comment (waldyrious) [inline src/compilers.coffee]: Unintended?

File: src/compilers.coffee [hunk 3]
```diff
@@ -168,6 +168,12 @@
   target: 'Java Bytecode'
   type: 'Intermediate'
   url: 'http://www.scala-lang.org/'
+,
+  name: 'Scala Natvie'
```
Comment (waldyrious) [inline src/compilers.coffee]: Typo: "Natvie" --> "Native"

**Comment by mohd-akram** [comment]
Thank you for contributing, and thank you @waldyrious for reviewing!

**Pull request merged**